src/game/engine/techno.cpp

```diff
@@ -529,12 +573,56 @@ void TechnoClass::Per_Cell_Process(PCPType pcp)
 
 RadioMessageType TechnoClass::Receive_Message(RadioClass *radio, RadioMessageType message, target_t &target)
 {
+#ifdef GAME_DLL
+    DEFINE_CALL(func, 0x005614C0, RadioMessageType, TechnoClass *, RadioClass *, RadioMessageType, target_t &);
+    return func(this, radio, message, target);
+#else
     return RadioMessageType();
+#endif
 }
 
 BOOL TechnoClass::Revealed(HouseClass *house)
 {
-    return 0;
+#ifdef GAME_DLL
```

Comment:
No need to keep the wrapping if you have a complete function, it should be enabled so it can be tested and fixed if needed.